roles/provision-keycloak-apb/tasks/provision-keycloak.yml

```diff
@@ -134,14 +134,20 @@
       PASSWORD: "{{ ADMIN_PASSWORD }}"
       IS_SHARED: "{{ USE_SHARED_SERVICE }}"
 
-- k8s_v1_secret:
-    name: "{{ keycloak_secret_name }}"
-    namespace: "{{ namespace }}"
-    labels:
-      name: keycloak
-      mobile: enabled
-      serviceName: keycloak
-    string_data:
-      user_id: "{{ ADMIN_USERNAME }}"
-      user_name: "{{ ADMIN_USERNAME }}"
-      user_passwd: "{{ ADMIN_PASSWORD }}"
\ No newline at end of file
+- name: "Create keycloak secret template"
+  template:
+    src: secret.yml.j2
+    dest: /tmp/secret.yaml
+  when: _apb_service_instance_id is defined
+
+- name: "Create keycloak secret template for tests"
```

Comment:
test logic and test templates should not be added to provision, bind tasks

Reply:
This is needed for the build to pass as `_apb_service_instance_id` is not available when running `apb test`

Reply:
apb test should run test.yml playbook, and all templates and data should be stored accordingly. test logic cant be mixed with production